themes/bootstrap3/js/common.js

```diff
@@ -737,12 +737,10 @@ function phoneNumberFormHandler(numID, regionCode) {
     } else {
       valid = VuFind.translate('libphonenumber_invalid');
     }
-    $(phoneInput).siblings('.help-block.with-errors').html(valid);
-    $(phoneInput).closest('.form-group').addClass('sms-error');
+    phoneInput.setCustomValidity(valid);
```

Comment:
It looks to me like phoneNumberFormHandler is no longer used anywhere. Since this is a patch release, it does make sense to fix it and keep it for now, but I doubt it's really being used in practice. Should we deprecate it here and remove it in 11.0?

Reply:
Yes, that's correct. I marked it deprecated (after a bit of distracted fumbling).